src/fittable_models.jl

```diff
@@ -0,0 +1,79 @@
+
+
+"""
+    FM90(;c1 = 0.10, c2 = 0.70, c3 = 3.23, c4 = 0.41, x0 = 4.60, gamma = 0.9)(λ::Real)
+    FM90(;c1 = 0.10, c2 = 0.70, c3 = 3.23, c4 = 0.41, x0 = 4.60, gamma = 0.9)(λ::Quantity)
+    FM90(coeffs, x0=4.60, gamma=0.9)(λ::Real)
+    FM90(coeffs, x0=4.60, gamma=0.9)(λ::Quantity)
```

Comment:
To avoid duplications we can have
```suggestion
    FM90(;c1 = 0.10, c2 = 0.70, c3 = 3.23, c4 = 0.41, x0 = 4.60, gamma = 0.9)(λ::Union{Real, Quantity})
    FM90(coeffs, x0=4.60, gamma=0.9)(λ::Union{Real, Quantity})
```

Reply:
I agree, I also find the spaces around every = quite disruptive and overextend the length of the signature.

Reply:
I will remove the spaces in next commit after we decide about the parameter tally and the name of the new category.